### PR TITLE
Modify deploy script to handle new branching structure.

### PR DESCRIPTION
**Pull request by gmierz**
This patch changes the circleci deployment script to handle the new branch stucturing.

### CODE REVIEW

**Comment by gmierz** [comment]
@bqbn I've replaced the code that checks for the directory's existence to only check if the CODE_DIR variable is set. Then, in the aws calls, I changed the CODE_DIR local paths to `.` - can you confirm that this will do what I'm expecting? (That it will take all the code within the current branch's directory).

File: .utils/deploy.sh [hunk 1]
```diff
@@ -72,7 +71,7 @@ deploy_code() {
     --metadata "{${CSP}, ${HSTS}, ${TYPE}, ${XSS}, ${REFERRER}}" \
     --metadata-directive "REPLACE" \
     --acl "public-read" \
-    "$CODE_DIR/" "s3://$YTTEST_BUCKET/$CODE_DIR/"
+    "." "s3://$YTTEST_BUCKET/$CODE_DIR/"
```
Comment (bqbn) [inline .utils/deploy.sh]: I haven't tested it but we may need `./` here (with a trailing slash).

File: .utils/deploy.sh [hunk 2]
```diff
@@ -83,7 +82,7 @@ deploy_code() {
...
```
Comment (bqbn) [inline .utils/deploy.sh]: Same as the above.

File: .utils/deploy.sh [hunk 3]
```diff
@@ -93,7 +92,7 @@ deploy_code() {
     --metadata "{${CSPSTATIC}, ${HSTS}, ${TYPE}, ${XSS}, ${REFERRER}}" \
     --metadata-directive "REPLACE" \
     --acl "public-read" \
-    "$CODE_DIR/" "s3://$YTTEST_BUCKET/$CODE_DIR/"
+    "." "s3://$YTTEST_BUCKET/$CODE_DIR/"
```
Comment (bqbn) [inline .utils/deploy.sh]: Same as the above.

**Review by bqbn — APPROVED** [review]
r+wc.

Once this is merged into `raptor-2020` branch, I believe it'll trigger auto-pushing to -stage env. We can then verify how it works there.

**Comment by gmierz** [comment]
Thanks for the review @bqbn! I've made the changes and will merge it shortly.

**Comment by gmierz** [comment]
@bqbn looks great to me :) thanks for your help! https://yttest.prod.mozaws.net/2020/main.html

**Pull request merged**